docs/changelog.rst

```diff
@@ -18,6 +18,12 @@ to know:
 * Please don't package beets as a Python 3 application *yet*, even though some
   things may appear to work under Python 3.4 and later.
 
+UI Change:
```

Comment:
Good point. I usually massage the release notes quite a bit before doing the final release anyway, so don't worry about it too much.